db/seeds.rb

```diff
@@ -1,8 +1,455 @@
 # frozen_string_literal: true
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+
+STUDENTS_LENGTH = rand(20..40).to_i
+
```

Comment:
Some sort of console logger would be nice to have before and after creating your data.

For example:
```suggestion
puts 'Creating Students...'
...
puts 'Students created successfully...'
```
Visualizing this can help you identify potential errors moving foward with future implementations in your projects.